dataproc/quickstart/quickstart.py

```diff
@@ -125,4 +124,23 @@ def quickstart(project_id, region, cluster_name, job_file_path):
     operation.result()
 
     print('Cluster {} successfully deleted.'.format(cluster_name))
-    # [END dataproc_quickstart]
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(
+        description=__doc__,
+        formatter_class=argparse.RawDescriptionHelpFormatter,
+    )
+    parser.add_argument('--project_id', type=str,
```

Comment:
nit: No hanging indents
```suggestion
    parser.add_argument(
      '--project_id', type=str,
```